spec/models/update_lock_spec.rb

```diff
@@ -17,41 +17,111 @@
 
       it "does not raise an error" do
         expect {
-          subject.check_availability!(2)
+          subject.check_availability!({transmitted_at: "1"})
```

Comment:
I don't think the braces around the `transmitted_at` option are strictly necessary.